python/holidays/is-IS.py

```diff
@@ -0,0 +1,73 @@
+from dateutil.easter import EASTER_WESTERN
+
+from utils import SmartDayArrow
+from .holidays import Holiday, Locale
+
+
+class is_IS(Locale):
+    """
+    01-01: [NRF] Nýársdagur
+    3 days before Easter: [NRV] Skírdagur
+    2 days before Easter: [NRV] Föstudagurinn langi
+    Easter: [NRV] Páskadagur
+    1 day after Easter: [NRV] Annar dagur páska
+    39 days after Easter: [NRV] Uppstigningardagur
+    49 days after Easter: [NRV] Hvítasunnudagur
+    50 days after Easter: [NRV] Annar dagur hvítasunnu
+    05-01: [NF] Verkalýðsdagurinn
+    06-17: [NF] þjóðhátíðardagurinn
```

Comment:
I'm not sure about this, to be honest. It's commonly called _17. júní_ but I'd just assumed it had a more official name. _17. júní_, _þjóðhátíðardagurinn_, and _lýðveldisdagurinn_ are all used, but the last isn't common. I've left it as _þjóðhátíðardagurinn_ since that explains it a bit more, but _17. júní_ would be fine too.

I capitalised the "Þ" in https://github.com/flother/holidata/commit/9edbdab7c34c4c10229f4727d169df13d0586742.